### ISSUE TITLE
A question about the s3dis dataset

### ISSUE THREAD

**Issue by YOOOOOL** [leob03/PointNet_Self_Attention#1]
Thanks for your open source work, I want to use this network to try the effect on s3dis dataset, can you give me some suggestions?

**Comment by leob03** [comment]
Glad you enjoyed it! 
Unfortunately in this repo we have only provided the code for the object classificication Task. If you want to use our Self-Attention residual layers this won't be problematic, simply look at the model_w_att.py file. Altough if you want to perform Semantic Segmentation on the s3dis dataset you willl have to do your own implementation of the Segmentation (see pictures).

<img width="790" alt="Screenshot 2023-09-21 at 22 05 43" src="https://github.com/leob03/PointNet_Self_Attention/assets/87416230/45508629-1c0f-4600-b7fe-51b5a538b3e3">

(fyi I have also added some more informations through the readme file if you're interested).
